internal/ice/gatherer.go

```diff
@@ -68,7 +68,7 @@ func NewGatherer(
 		}
 	}
 
-	candidateTypes := []ice.CandidateType{}
+	candidateTypes := make([]ice.CandidateType, 0)
```

Comment:
This change has to do with Jetbrain's Goland go linting complaining that this is better.